defaults/main.yml

```diff
@@ -3,6 +3,7 @@ node_exporter_version: 0.17.0
 node_exporter_web_listen_address: "0.0.0.0:9100"
 
 node_exporter_textfile_dir: "/var/lib/node_exporter"
+node_exporter_destination: "/usr/local/bin/node_exporter"
```

Comment:
This is static on purpose as we are trying to follow dir structure defined in `hier(7)`.

Reply:
In `hier`?

Reply:
`man hier` manual for a layout of linux system directory tree

Reply:
@paulfantom ah, I had seen `node_exporter` in `/opt` on a few systems. Do you mind if I keep this? Since the default is what you need.

Reply:
Yes, we put it into `/opt` in the beginning, then there was a transition period which took couple of months, where we had this configurable and did some tricks to cleanup after ourselves. And now it is over and I don't see any reason to go back.

Reply:
@paulfantom I'll see if I can finish this PR this week and remove it then.